.github/workflows/release.yaml

```diff
@@ -0,0 +1,55 @@
+name: Make release with Build binaries
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  build-binary:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: PyInstaller win64
+        id: win64
+        uses: action-python/pyinstaller-py3.8@win64
+        with:
+          path: ./
+          requirements: "./requirements_nt.txt"
+      - name: PyInstaller amd64
+        id: amd64
+        uses: action-python/pyinstaller-py3.8@amd64
+        with:
+          path: ./
+          requirements: "./requirements.txt"
+      - name: Create Release
```

Comment:
Looks like this approach is good only when release is triggered by push of tag from git client. If release is already created using web UI, probably we could just edit the created release? Also I suggest to think about possible unification of this workflow and Publish Artifacts workflow as they might potentially be triggered by the same `create release` event.

Reply:
Now you can use the web UI to create releases